recipes/instaseis/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "7fece415cca6c70e575531a463c0d5cb6d30f59740d7a113a02d584775a8b0c1" %}
+
+package:
+  name: instaseis
+  version: {{ version }}
+
+
+source:
+  fn: instaseis-{{ version }}.zip
+  url: https://pypi.io/packages/source/i/instaseis/instaseis-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+    - numpy
+    - h5py
+    - obspy
+    - future
+    - tornado
+    - requests
+    - jsonschema
```

Comment:
The reason I'm proposing this is I forgot that this NumPy 1.0 problem with `conda-build-all` is not only not fixed, but was not raised at all. 😞 This sort of change is a decent workaround for the problem.

Have now at least raised the upstream issue ( https://github.com/SciTools/conda-build-all/issues/56 ). A decent workaround for `conda-build-all` is trivial. However, I'm not sure that it isn't a `conda-build` issue at the end of the day.